docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -132,39 +136,39 @@ Here you can specify port ranges that should be filtered.
 
 #### Log removed HTML events
 
-If enabled, AdGuard records blocked HTML elements in the filtering log.
+If enabled, AdGuard records blocked HTML elements in *Recent activity*.
 
 #### Scriplet debugging
 
-If you need to activate debugging of scriptlets, enable this feature. Then there will be messages in the browser log that some scriplet rules have been applied.
+This options displays information about applied scriptlet rules to the browser log.
 
 #### Excluded apps
 
-Here you can list package names and UIDs that you want to exclude from AdGuard protection.
+Here you can list package names and UIDs that you want to exclude from AdGuard's protection.
 
 #### QUIC bypass packages
 
 Here you can specify package names for which AdGuard should bypass QUIC traffic.
 
 #### Reconfigure Automatic proxy when network changes
 
-Enable this setting if you want the protection to restart to reconfigure the automatic proxy settings when the device connects to another network. The state of this setting affects operation only if the current routing mode is Automatic proxy.
+This option restarts protection to reconfigure the automatic proxy settings when the device connects to another network. It is only applied if *Automatic proxy* is selected in *Routing mode*.
```

Comment:
If this is enabled, the AdGuard protection will restart to reconfigure the automatic proxy settings when your device connects to another network. This setting only applies when *Routing mode* is set to *Automatic proxy*.